src/js/components/utils/detailPageLayout.tsx

```diff
@@ -167,16 +170,16 @@ const DetailPageLayout = ({
             />
           )}
           {sidebar}
-        </div>
-        <div
+        </aside>
+        <main
```

Comment:
Before today I didn't know about the `main` tag. This is a great learning moment for me. Making sure every page has one `main` tag seems like a best practice.

Reply:
Good semantic HTML markup is the most effective tool we have for accessibility. Here's a quick little article that covers the common semantic elements. 

https://uxdesign.cc/semantic-html-the-foundation-of-web-accessibility-e5bbecad7c17?gi=7398cc5a27d7